src/app/src/components/QuizHome.js

```diff
@@ -13,23 +21,73 @@ const StyledQuizHome = styled(Flex)`
     justify-content: center;
 `;
 
+const StyledFinalQuizState = styled(StyledQuizHome)`
+    margin-top: 15%;
+`;
+
+const StyledScore = styled(Header)`
+    font-weight: 900;
+    font-size: 8.5rem;
+`;
+
+const StyledButton = styled(Button)`
+    margin: 1rem;
+`;
+
 class QuizHome extends Component {
     componentWillUnmount() {
         const { dispatch } = this.props;
-        dispatch(clearQuizResults());
+        dispatch(clearQuizScore());
     }
 
     render() {
-        const { dispatch, finalQuizState } = this.props;
+        const { dispatch, quizScore, secondsToCompleteQuiz } = this.props;
+
+        let bonusPoints = 0;
+        if (secondsToCompleteQuiz < 31) {
+            bonusPoints += 200;
+        } else if (secondsToCompleteQuiz < 61) {
+            bonusPoints += 150;
+        } else if (secondsToCompleteQuiz < 121) {
+            bonusPoints += 100;
+        }
 
-        const quizHomeState = finalQuizState ? (
-            <Button mt='compact' onClick={() => dispatch(clearQuizResults())}>
-                Try again
-            </Button>
+        const quizHomeState = quizScore ? (
+            <StyledFinalQuizState>
+                <Header variant='small'>YOU GOT A SCORE OF</Header>
+                <Flex alignItems='baseline'>
+                    <StyledScore>{quizScore + bonusPoints}</StyledScore>
+                    <Text variant='large'>/500 points</Text>
```

Comment:
This is more of a design thing, but I found it kind of confusing when my score was higher than 500. Consider dropping the `/500` or add something that shows the time bonus given.

Reply:
Yes ! Soon that will be the case. The time bonus message will be a Lottie animation.